pages/downloads/0.9.x-release-notes/[...slug].js

```diff
@@ -19,14 +19,10 @@
 import fs from "fs";
 import matter from "gray-matter";
 import React from "react";
-import ReactMarkdown from "react-markdown";
 import { Container, Col, Button, Offcanvas } from "react-bootstrap";
-import remarkGfm from "remark-gfm";
-import rehypeRaw from "rehype-raw";
 import Image from "next-image-export-optimizer";
 import Head from "next/head";
-import { getHighlighter, setCDN } from "shiki";
-
+import { setCDN } from "shiki";
```

Comment:
Remove `setCDN`.  and `setCDN("https://unpkg.com/shiki/");` in line 31. There are not used anymore. Check other slugs too.